### ISSUE TITLE
Support evaluation of local PyTorch models

### ISSUE THREAD

**Issue by saattrupdan** [alexandrainst/AIAI-eval#18]
This should include the following:

1. Allow specifying the (relative) path to a local `bin` file with the model, input the `torch.nn.Module` instance itself, if the scripting approach to the package is used.
2. Useful error messages if various things could not be handled.
3. Some extended logging (if `verbose` is enabled), displaying inferred elements of the model, such as label enumeration, output type (logits/probabilities) etc.
